tsconfig.json

```diff
@@ -30,5 +30,5 @@
     "lib": ["es2017", "dom"],
     "typeRoots": ["./node_modules/@types", "./src/typings"]
   },
-  "exclude": ["dist", "node_modules", "sandbox.ts", "test.ts"]
+  "exclude": ["dist", "node_modules", "sandbox.ts", "test.ts", "src-temp"]
```

Comment:
Do you still need `src-temp` here?

Reply:
Yeah. The new typedoc version just tries to build everything it finds